src/yashCJun20/CountSpecialChar.java

```diff
@@ -0,0 +1,28 @@
+package yashCJun20;
+
+public class CountSpecialChar {
+	
+	void cntSpclChar (String name) {
+		String uname;
+		
+		uname=name.replace("[", "");
```

Comment:
From next time you should write it in one statement.

name.replace("[","").replace("{","") and son